plasmapy/classes/plasma_base.py

```diff
@@ -0,0 +1,26 @@
+from collections import OrderedDict
+from abc import ABC
+
+# GenericPlasma subclass registry
+PLASMA_CLASSES = OrderedDict()
```

Comment:
Do we really need to use an `OrderedDict` here or a simple `dict` object would suffice? SunPy seems to use `OrderedDict` but I am not sure about its usefulness (plus it is slower).

Reply:
Given you are Python 3.6+ you do not need `OrderedDict` here. Also I don't think it really matters at all.

Reply:
Ok, I guess then we should probably stick with `dict` atleast for now.